src/commands/SetTrackInfoCommand.cpp

```diff
@@ -357,6 +360,14 @@ void SetTrackVisualsCommand::PopulateOrExchange(ShuttleGui & S)
       S.Optional( bHasSpectralSelect ).TieCheckBox( XXO("Spectral Select"),    bSpectralSelect);
    }
    S.EndMultiColumn();
+   S.StartMultiColumn(3, wxEXPAND);
+   {
+      S.SetStretchyCol( 2 );
+      auto schemes = SpectrogramSettings::GetColorSchemeNames();
+      S.Optional( bHasSpecColorScheme).TieChoice( XXO("Spectro. Color scheme:"), mSpecColorScheme,
```

Comment:
Also for string freeze, it's too late now to add new user-visible strings -- or else they won't translate from English.  So change this to reuse the same string as in the preferences dialog:

`XC("Sche&me", "spectrum prefs")`